SurveyTests/Sources/Mocks/Home/HomeUseCaseMock.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  HomeUseCaseMock.swift
```

Comment:
You can try this tool to help you create theses mocks https://github.com/krzysztofzablocki/Sourcery/blob/master/guides/Mocks.md. Take a look at our MVVM demo for example.

Reply:
Yeah, I will try it next time, thank you. 😄